### ISSUE TITLE
Improve chronicle queue documentation

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Queue#650]
Please document in the .adoc the following: 

```
1.BufferMode
None,
Copy,
Asynchronous
              
2.bufferCapacity
3.blockSize
4.indexSpacing
5.indexCount
6.Pauser
7.strongAppenders
```

**Comment by dpisklov** [comment]
@RobAustin did you put the info from the email yesterday to adoc or shall I?

**Comment by RobAustin** [comment]
nope, I added it to the Javadoc, yes, we should cover most if not some of this on adoc. @dpisklov yes please could you add it.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.19.3](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.19.3), [BOM-2.19.45](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.45)
